start_pose_num: 1
end_pose_num: 51 
headYaw_min: 0
headYaw_max: 0.8
headYaw_step: 0.4
headPitch_min: -0.5
haedPitch_max: 0.2
headPitch_step: 0.33
find_marker: true
move_marker_to_corners: false
marker_color_r: 0
marker_color_g: 255
marker_color_b: 0